docs/docs/gatsby-style-guide.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Gatsby Style Guide
+---
+
+The Gatsby community is building a massive, searchable "Gatsby Guides." This reference tool will eventually include thousands of articles that cover all topics relevant to people building on Gatsby - all written to be easily understood by the awesome community of people who love building with Gatsby!
+
+This repo is where we plan, write, and maintain these Guide articles, which we then host at Gatsby Guides.
+
+## Table of Contents
+- [What are Guide articles?](#what-are-guide-articles?)
+- [What can I write an article about?](#what-can-i-write-an-article-about?)
+- [How to contribute](#how-to-contribute)
+- [Running the Guide locally on your own computer](#running-the-guide-locally-on-your-own-computer)
+- [Article style guide](#article-style-guide)
+
+# What are Guide articles?
+
+Guide articles can cover a broad range of topics. Please see the following examples: [tutorials](https://www.gatsbyjs.org/tutorial/part-one/), [code documentation](https://www.gatsbyjs.org/docs/browser-apis/), [explanatory articles](https://www.gatsbyjs.org/docs/prpl-pattern/), and [focused guides](https://www.gatsbyjs.org/docs/add-404-page/).
+
+# What can I write an article about?
+
+We welcome your help writing these articles. You don't have to be an expert in a topic to write about it - this entire Guide is open source, so even if you make a mistake, another contributor will eventually correct it.
+
+If you’d like to help write an article, find a stub article in the Gatsby Guides, write the article, then open a pull request (PR) in this repo to replace the stub with your article.
+
+If you can't find a stub about the topic you'd like to write about, you can open a PR that creates the stub and includes your draft article. Feel free to ask us questions if you're not sure where to put a new article in the directory structure.
+
+Before you begin writing, make sure to read the article style guide below.
+
+# How to contribute
+
+You can create a PR with your draft article (or edits on an existing article) in two ways:
+
+1. The easiest method is to use the GitHub interface. Watch the video demonstration or follow the steps below it: [insert video if needed. Is this necessary?]
+2. Go into the ["docs" folder](gatsbyjs/gatsby/docs) and find the article stub you'd like to write or edit. All stubs will be in an index.md file.
```

Comment:
this should be the "full" link https://github.com/gatsbyjs/gatsby/tree/master/docs/docs since people will be reading this on gatsbyjs.org so your relative link won't work.